WORKSPACE

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The StableHLO Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Workspace for StableHLO."""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+SKYLIB_VERSION = "1.3.0"
+
+http_archive(
+    name = "bazel_skylib",
+    sha256 = "74d544d96f4a5bb630d465ca8bbcfe231e3594e5aae57e1edbf17a6eb3ca2506",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/{version}/bazel-skylib-{version}.tar.gz".format(version = SKYLIB_VERSION),
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/{version}/bazel-skylib-{version}.tar.gz".format(version = SKYLIB_VERSION),
+    ],
+)
+
+LLVM_COMMIT = "afca08a567e3803aeb0c32385e62c84bec58ef46"
+
+LLVM_SHA256 = "c08c8f6cfec9134dc28464c1d660958cf25bc2186e8459a5d96205e5bd7c4975"
```

Comment:
Any way to protect ourselves from this value going stale? Can we add CI for mismatch detection?

Reply:
LLVM_COMMIT will always be in sync with LLVM_SHA256 if that's what you're wondering. Lmk if you mean to ask something else.

Reply:
I meant "if workspace llvm_commit ever diverges from llvm_version.txt in a CI job, we should fail" 

Let's talk more about this offline, I remember you saying something about running this on HEAD-1, maybe I didn't understand that